script/Arcueid/Day 7/07_00_ARC07_1_2.txt

```diff
@@ -188,12 +204,13 @@ My mind goes blank.
 [sha:d2cc8ccc777df074ec4b7c348cd4b8faf56ee19f]{
 -- Page 147, Offset 16942.
 -- つまり……アルクェイドはいま、遠野の屋敷に？
-In other words... right now, Arcueid is in the mansion?
+Does this mean... At this very moment, Arcueid is at the Tohno mansion?
 }
 [sha:715faa8ce32044382de44b5bc9324d8d3b73c922]{
 -- Page 147, Offset 16943.
 -- 思考は最悪の事態をたやすく理解し、そして理解を拒んだ。
-My thoughts quickly turn to the worst case scenario, which I refuse to comprehend.
+I quickly grasp the ramifications of how poorly this could go,
```

Comment:
this is a mis-TL and the original is much closer.
> そして理解を拒んだ。

would be
> but I _refused_ [to grasp the ramifications of] my reasoning

Reply:
Yeah, this one was on purpose since I felt it captured the intent while also reading a lot more sensibly than 
> My thoughts easily comprehend the worst case scenario, and then refuse to comprehend it.

Alternates that are maybe closer:
> Thoughts of what could happen in the worst case scenario leap into my mind, but my brain desperately blots them out.
> I can easily see this going horribly wrong, but my brain refuses to even comprehend the possible ramifications.

Reply:
my preference is towards " I can easily see this going horribly wrong, but my brain refuses to even comprehend the possible ramifications."